src/schema/tags.ts

```diff
@@ -76,5 +81,22 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (source, args, ctx): Promise<GQLTagSearchResults> => {
```

Comment:
Why do we return the response as search results?

Reply:
Wanted to reuse the type, but I guess it does not hurt to add another one.

Reply:
Yes, I think it's better to define a new type for that. Of course we can keep it 99% compatible

Reply:
Yeah, updated already [48dbfdc](https://github.com/dailydotdev/daily-api/pull/1480/commits/48dbfdcf110a2618671c4d83e91b04ddec2a8103)